spec/reckon/money_column_spec.rb

```diff
@@ -35,11 +35,23 @@
       Reckon::MoneyColumn.new( ["1.00", ""] ).merge!( 
         Reckon::MoneyColumn.new( ["", "-2.00"] ) ).should == [ 
           Reckon::Money.new( 1.00 ), Reckon::Money.new( -2.00 ) ]
-    end
+      Reckon::MoneyColumn.new( ["1.00", "0"] ).merge!( 
+        Reckon::MoneyColumn.new( ["0", "-2.00"] ) ).should == [ 
+          Reckon::Money.new( 1.00 ), Reckon::Money.new( -2.00 ) ]
+      Reckon::MoneyColumn.new( ["AB1.00C", ""] ).merge!( 
+        Reckon::MoneyColumn.new( ["", "AB-2.00C"] ) ).should == [ 
+          Reckon::Money.new( 1.00 ), Reckon::Money.new( -2.00 ) ]
+      Reckon::MoneyColumn.new( ["AB1.00C", "AB0C"] ).merge!( 
+        Reckon::MoneyColumn.new( ["AB0C", "AB-2.00C"] ) ).should == [ 
```

Comment:
Do you have an example CSV that has numbers in this format?  It might be worth adding above... or is this the Suntrust one?  I just don't see why that one was having issues.